interfacesegregationprinciple/HinhChuNhat.java

```diff
@@ -0,0 +1,18 @@
+package solid.interfacesegregationprinciple;
+
+public class HinhChuNhat implements IMul {
```

Comment:
Please use English only. Check the other places as well! Thx

Reply:
I have adjusted.